### PR TITLE
Update cades build script

### PR DESCRIPTION
**Pull request by jtkrogel**
Due to recent changes in the CADES environment, the build script ceased to work.  This PR updates the build script to match the build recipe used in the CADES qmcpack module.

### CODE REVIEW

**Comment by prckent** [comment]
Thank for improving this.

File: config/build_ornl_cades.sh [hunk 1]
```diff
@@ -7,28 +7,36 @@
...
 # Set environment variables
 export FFTW_HOME=$FFTW_DIR/..
+export BOOST_ROOT=$BOOST_DIR
 
 CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
              -DCMAKE_CXX_COMPILER=mpicxx \
              -DCMAKE_C_FLAGS=-xCOMMON-AVX512 \
-             -DCMAKE_CXX_FLAGS=-xCOMMON-AVX512"
+             -DCMAKE_CXX_FLAGS=-xCOMMON-AVX512 \
+             -DHAVE_MKL=1"
```
Comment (ye-luo) [inline config/build_ornl_cades.sh]: Please use -DENABLE_MKL=1 instead of -DHAVE_MKL=1.
See the logic here #1436

Comment (ye-luo) [reply config/build_ornl_cades.sh]: Did you find our FindMKL not working as expected even with intel compiler?

Comment (jtkrogel) [reply config/build_ornl_cades.sh]: Switched to -DENABLE_MKL=1.  Seems to work fine.

Comment (ye-luo) [reply config/build_ornl_cades.sh]: I’m still wondering why you need it. It should not be needed when intel compiler is in use.  What happens if you don’t put it.

Comment (jtkrogel) [reply config/build_ornl_cades.sh]: Also works fine without ENABLE_MKL.  Removing it now.

**Comment by ye-luo** [comment]
Okay to test

**Pull request merged**